Makefile

```diff
@@ -260,7 +260,27 @@ release-manager-snapshot: release
 .PHONY: release-manager-release
 release-manager-release: release
 
+JAVA_ATTACHER_VERSION:=1.26.0
+JAVA_ATTACHER_JAR:=apm-agent-attach-cli-$(JAVA_ATTACHER_VERSION).jar
```

Comment:
You can use the smaller `cli-slim.jar` that doesn't bundle the actual agent. That works because the attacher is always invoked via [`--download-agent-version`](https://www.elastic.co/guide/en/apm/agent/java/current/setup-attach-cli.html#setup-attach-cli-usage-options) which download the actual agent from maven central.